docs/pages/_app.js

```diff
@@ -140,7 +144,32 @@ function PersistState() {
     window.ga('set', 'dimension2', options.userLanguage);
   }, [options.userLanguage]);
 
-  return null;
+  React.useEffect(() => {
+    /**
+     * @type {null | MediaQueryList}
+     */
+    let matchMedia = null;
+
+    /**
+     * Based on https://developer.mozilla.org/en-US/docs/Web/API/Window/devicePixelRatio#Monitoring_screen_resolution_or_zoom_level_changes
+     * Adjusted to track 3 or more different ratios
+     */
+    function trackDevicePixelRation() {
+      window.ga('set', 'dimension3', window.devicePixelRatio);
```

Comment:
Already added to google-analytics.

Reply:
I'm curious to see if we will get a long tail of values with decimals. Say, 2.33, 2.19, etc.

Reply:
It seems that we need to round the number. Can you process the information in this [GA view](https://analytics.google.com/analytics/web/#/report/visitors-userdefined/a106598593w159022482p160376982/_u.date00=20200527&_u.date01=20200527&explorer-table.secSegmentId=analytics.customDimension3&explorer-table.plotKeys=%5B%5D&explorer-table.rowCount=250&explorer-table-dataTable.sortColumnName=analytics.customDimension3&explorer-table-dataTable.sortDescending=false)

<img width="1063" alt="Capture d’écran 2020-05-27 à 21 59 52" src="https://user-images.githubusercontent.com/3165635/83066578-8c0c9f80-a065-11ea-9196-152b63594d2c.png">